### PR TITLE
[Scheduler] generate initial client

### PR DESCRIPTION
**Pull request by TheRoyalTnetennba**
* Generate the gapic scheduler client
* Add generating helpers to alias class methods to instance methods
* Add tests for generated helpers
* Update .rubocop.yml as per https://github.com/googleapis/google-cloud-ruby/pull/2721

### CODE REVIEW

File: google-cloud-scheduler/lib/google/cloud/scheduler/v1beta1/cloud_scheduler_client.rb [hunk 1]
```diff
@@ -0,0 +1,621 @@
...
+              options: nil,
+              &block
+            req = {
+              name: name
+            }.delete_if { |_, v| v.nil? }
+            req = Google::Gax::to_proto(req, Google::Cloud::Scheduler::V1beta1::RunJobRequest)
+            @run_job.call(req, options, &block)
+          end
+
+          def job_path *args
```
Comment (blowmage) [inline google-cloud-scheduler/lib/google/cloud/scheduler/v1beta1/cloud_scheduler_client.rb]: I would rather see these methods added via a helpers file and include documentation.

File: google-cloud-scheduler/lib/google/cloud/scheduler/v1beta1/helpers.rb [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          
+          # Alias for Google::Cloud::Scheduler::V1beta1.job_path.
+          # @param args [Array<String>]
+          # @return [String]
+          def job_path *args
+            self.class.job_path *args
+          end
+          
+          # Alias for Google::Cloud::Scheduler::V1beta1.project_path.
+          # @param args [Array<String>]
```
Comment (blowmage) [inline google-cloud-scheduler/lib/google/cloud/scheduler/v1beta1/helpers.rb]: Then non-generated methods document the `project`, `location`, and `job` arguments. If i had to choose between documenting the arguments and generating this file I would choose documenting the arguments.

Comment (TheRoyalTnetennba) [reply google-cloud-scheduler/lib/google/cloud/scheduler/v1beta1/helpers.rb]: Good call. added

File: google-cloud-scheduler/synth/helpers.rb.erb [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+module Google
+  module Cloud
+    module Scheduler
+      module <%= version.capitalize %>
+        class CloudSchedulerClient
+
+<%= 
+  helpers = []
+  class_methods.each do |method|
+    helpers << "# Alias for Google::Cloud::Scheduler::#{version.capitalize}.#{method}."
```
Comment (blowmage) [inline google-cloud-scheduler/synth/helpers.rb.erb]:
```ruby
helpers << "# Alias for Google::Cloud::Scheduler::#{version.capitalize}::CloudSchedulerClient.#{method}."
```

Comment (TheRoyalTnetennba) [reply google-cloud-scheduler/synth/helpers.rb.erb]: Thanks! Fixed

**Pull request merged**